books/free-programming-books-fr.md

```diff
@@ -268,6 +268,7 @@
 
 * [Apprendre à programmer avec Python](http://inforef.be/swi/python.htm) - Gerard Swinnen
 * [Introduction à la programmation](https://self-learning.info.ucl.ac.be/index/info1-exercises) (Inscription gratuite sur le site. Pour réaliser les exercices sur INGInious.org, créez-vous un compte gratuitement et liez ensuite votre compte self-learning à votre compte INGInious. )
+* [Introduction au Machine Learning](https://www.youtube.com/watch?v=nDnEKGxQCdA&list=PL0mGkrTWmp4ugGM9fiZjEuzMFeKD6Rh5G) -DSN, Data Scientist Network
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

Reply:
```suggestion
* [Python Programming in French](https://www.youtube.com/playlist?list=PL0mGkrTWmp4ugGM9fiZjEuzMFeKD6Rh5G) - Data Scientist Nigeria
```